### ISSUE TITLE
dataset zip cannot be unzipped by standard macOS unzip due to required PK compat v4.5

### ISSUE THREAD

**Issue by corneliusroemer** [ncbi/datasets#414]
When unpacking a downloaded NCBI dataset zip folder with standard unzip on macOS, I get the following error:

```
Archive:  results/ncbi_dataset.zip
   skipping: README.md               need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/data_report.jsonl  need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/genomic.fna  need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/virus_dataset.md  need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/dataset_catalog.json  need PK compat. v4.5 (can do v2.1)
```

I don't remember having this problem in the past, it might be a recent change?

How are you creating the zip folder, have you changed the program with which this happens recently? It would be nice if people could just use standard OS software to unzip.

Repro (on macOS):

```sh
$ datasets download virus genome taxon 3052462 --no-progressbar --filename results/ncbi_dataset.zip --api-key None

$ unzip -o results/ncbi_dataset.zip -d results
Archive:  results/ncbi_dataset.zip
   skipping: README.md               need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/data_report.jsonl  need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/genomic.fna  need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/virus_dataset.md  need PK compat. v4.5 (can do v2.1)
   skipping: ncbi_dataset/data/dataset_catalog.json  need PK compat. v4.5 (can do v2.1)
   skipping: md5sum.txt              need PK compat. v4.5 (can do v2.1)
```

I'm using dataset version 16.31.0 and macOS 15

Workaround: use `zip` instead of `unzip`

**Comment by corneliusroemer** [comment]
Actually - it seems that standard macOS unzip works - it's the conda-forge unzip that doesn't!

**Comment by mtntsuchiya** [comment]
Hi Cornelius,
Thank you for reporting and for the update! I'm glad you found a solution, or at least the issue. 
Please let us know if we can help in any way.
Best,
Mirian

**Comment by corneliusroemer** [comment]
Thanks @mtntsuchiya! 

While I've figured out that one cause seems to be that conda-forge's unzip has less PK compat support than mac's default unzip, it's still surprising that we hit this only now - we did local testing without issue, this only happened recently.

Is it possible that the recent ZIP change in version 16.31.0 might have contributed to this? It still seems likely that the zip folder used to only require 2.1 or lower for unzipping.

See https://github.com/conda-forge/unzip-feedstock/issues/16 for the conda-forge unzip issue I opened.
